sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3635,7 +3640,8 @@ piEnqueueKernelLaunch(pi_queue Queue, pi_kernel Kernel, pi_uint32 WorkDim,
 pi_result piEventCreate(pi_context Context, pi_event *RetEvent) {
   size_t Index = 0;
   ze_event_pool_handle_t ZeEventPool = {};
-  ZE_CALL(Context->getFreeSlotInExistingOrNewPool(ZeEventPool, Index));
+  Context->getFreeSlotInExistingOrNewPool(ZeEventPool, Index);
```

Comment:
The ze_result_t return value of this method is always ignored.
Shouldn't the method return void then? Or changing the return from ze_result_t to void breaks backward compatibility?

Reply:
Added error check